notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -62,6 +62,49 @@ sexual debut [Charly was deflowered by his former teacher Alice.]
 master and pet [Charly was heartbroken went his pet mouse Algernon died.]
 
 
+movie: Mission to Mars (2000)
+=============================
+
+:: Title
+Mission to Mars
+
+:: Description
+Mission to Mars is a 2000 American science fiction adventure film directed by
+Brian De Palma, written by Jim Thomas, John Thomas, and Graham Yost, and
+inspired by Disney's theme park attraction of the same name.
+
+Synopsis: When the first manned Mars exploration mission goes awry, a rescue
+mission is dispatched to return any survivors to Earth.
+
+:: Date
+2000-03-10
+
+:: References
+https://en.wikipedia.org/wiki/Mission_to_Mars
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+interplanetary space travel [The first half of the film depicts a pioneering manned spaceflight from Earth to Mars.]
+human vs. inexplicable adversity [The astronauts were at a loss for how to explain the sudden appearance of a large humanoid face on the surface of Mars. The second half of the film concerned the astronauts' efforts to get to the bottom of what it was and how it had gotten there in the first place.]
```

Comment:
Your comment suggests that they were merely solving a mystery. Wherein was the adversity (problematic happenings)? I guess it raises the question of what theme to use when there is simply a non-threatening enigma to solve. But I somehow suspect in this case that your comment is just incomplete...